__tests__/workers/notifications.ts

```diff
@@ -168,6 +168,163 @@ it('should NOT add devcard unlocked notification if user had already reached the
   expect(actual).toBeUndefined();
 });
 
+describe('squad featured updated notification', () => {
+  const baseMember = {
+    userId: '1',
+    sourceId: 'squad',
+    referralToken: 'rt1',
+  };
+
+  const squad = {
+    ...sourcesFixture[0],
+    type: SourceType.Squad,
+    flags: { featured: true },
+  };
+
+  beforeEach(async () => {
+    await saveFixtures(con, User, badUsersFixture);
+    await con
+      .getRepository(Source)
+      .update({ id: 'a' }, { type: SourceType.Squad });
+    await con.getRepository(SourceMember).save({
+      role: SourceMemberRoles.Admin,
+      sourceId: 'a',
+      userId: '1',
+      referralToken: 't1',
+    });
+    await con.getRepository(SourceMember).save({
+      role: SourceMemberRoles.Admin,
+      sourceId: 'a',
+      userId: '2',
+      referralToken: 't2',
+    });
+    await con.getRepository(SourceMember).save({
+      role: SourceMemberRoles.Moderator,
+      sourceId: 'a',
+      userId: '3',
+      referralToken: 't3',
+    });
+    await con.getRepository(SourceMember).save({
+      role: SourceMemberRoles.Member,
+      sourceId: 'a',
+      userId: '4',
+      referralToken: 't4',
+    });
+    await con.getRepository(SourceMember).save({
+      role: SourceMemberRoles.Blocked,
+      sourceId: 'a',
+      userId: 'low-score',
+      referralToken: 't5',
+    });
```

Comment:
you can do single `save` for all of those?

Reply:
Yeah, was copy-pasting it and did not give the saving part much thought as it was a test. I can update this.

Reply:
also if you test needs specific order of source members add `createdAt` manually, see `__tests__/integrations.ts` for example

Reply:
Ah, date should not be a problem since I manually `find` the relevant items. Thanks for the reference.